js/containers/portal-dashboard/portal-dashboard-app.tsx

```diff
@@ -67,6 +70,11 @@ class PortalDashboardApp extends React.PureComponent<IProps, IState> {
     if (initialLoading && !isFetching && prevProps.isFetching) {
       this.setState({ initialLoading: false });
     }
+
+    const studentAnswersRef = this.studentAnswersComponentRef.getStudentAnswersRef();
+    if (studentAnswersRef && this.state.scrollLeft !== studentAnswersRef.scrollLeft * -1) {
+      this.setState({ scrollLeft: studentAnswersRef.scrollLeft * -1 });
```

Comment:
This handles the case where the scroll changes because content has been collapsed (activity button is pressed) rather than the scroll bar being moved.